marketplaceApp/db/migrate/20201102185838_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
if we do a third-party auth token thing, where do we store that info? or is it done automatically? i think we can edit the db as we go along so it's not a big deal atm but I'm just curious

Reply:
that's a good point! depending on how the SSO works, we might end up with a third table that would relate to the user model (or we can add extra fields).

Reply:
I am gonna get working on the SSO hopefully wednesday or thursday, I can let you know if anything tricky involving the db comes up but hopefully its straightforward